### ISSUE TITLE
allow values to be set to nil (or "")

### ISSUE THREAD

**Issue by minshallj** [chrislee35/configparser#4]
```
[some_section]
foo=
```

When this is parsed, it will be `{}`. I'd sort of like to have it look more like `{"some_section"=>{"foo"=>""}}`

This would match the default behavior in the python config parser. Using 
```python
cfg = ConfigParser.RawConfigParser()
cfg.read('/tmp/foo.cfg')
cfg.get('some_section', 'foo')
=> ''
```

**Comment by chrislee35** [comment]
Thank you so much for spotting this for me.  
This is now tested and patched in 0.1.5.  Please test and confirm.

**Comment by minshallj** [comment]
Awesome, thanks a bunch. It's working for me (installed the gem version 0.1.5).
